core/src/ingestion.ts

```diff
@@ -1,11 +1,11 @@
 import { MikroORM } from '@mikro-orm/postgresql';
 
 import { logger } from './logger'
-import { NatsMessage } from './nats'
 import { Change } from "./entities/Change"
 import { FetchedRecord } from './fetched-record'
 import { FetchedRecordBuffer } from './fetched-record-buffer'
 import { stitchFetchedRecords } from './stitching'
+import { Consumer, JsMsg } from 'nats';
```

Comment:
Nit: separating local imports similarly to https://github.com/BemiHQ/bemi/pull/4/files#r1622386701 here and in `fetched-record.ts`

Reply:
This is updated.